src/media/css/elements--nav.styl

```diff
@@ -212,7 +212,10 @@ mkt-nav-child {
     transition: unquote($nav--transition);
     width: $nav--width;
     z-index: $zindex--nav-hamburger-child;
-
+    
+    +rtl() {
+        padding: 0 22px 0 0;
```

Comment:
Can you update the other padding rule to be `0 22px` instead of overriding for RTL please.